articles/quickstart/webapp/any/01-login.md

```diff
@@ -0,0 +1,167 @@
+5. Your application calls the Auth0 /oauth/token endpoint with the Authorization Code, asking to exchange it with an access_token.
+6. Auth0 authenticates the web app, validates the Authorization Code, and responds back with the token.
+7. The web application decodes and verifies the token, which confirms the user's identity.
+8. Your application uses sessions or cookies to persist a user login.
+
+## Register Your App as a Client
+
+Go to the [Clients](${manage_url}/#/clients) page from the [Auth0 Dashboard](${manage_url}). Edit the default client or create a new client by clicking **Create Client**.
+
+Name your new application, and set the Client Type to **Regular Web Applications** as the **Client Type**.
+
+### Set Allowed Callback URI
+
+When your app redirects a user to Auth0, the querystring will include a callback URI. After authentication completes, the user will be redirected to this URI.
+
+${include('../_callbackRegularWebApp')}
+
+While you are editing your Client settings, update your Allowed Callbacks list to include the redirect URI you plan to use.
+
+### Enable OIDC Conformance
+
+Before you leave Client Setting, click on **Show Advanced Settings**. Go to the **OAuth** tab and make sure tha the **OIDC Conformant** switch is enabled.
+
+Save your settings.
+
+## Add Users
+
+Before users can login, you need to [add some registered users](/user-profile).
+
+## Create a Log In Link
+
+Your users will log in to your site using the Auth0 authentication portal at the URI `https://${account.namespace}/authorize`.
+
+To facilitate this, you will provide a link from your site with the following query parameters:
+
+| Parameter | Value |
+|:----------|:---------|
+| response_type | `code` |
+| scope | `openid` |
+| redirect_uri | The URI that the user will be redirected to after authentication (`${account.callback}`)|
+
+```html
+<a href="https://${account.namespace}/authorize?response_type=code&scope=openid&redirect_uri=${account.callback}">Log In</a>
+```
+
+## Handle the Callback
+
+
+
+After the user has authenticated, Auth0 will redirect the user to the URI specified in the `redirect_uri` parameter of the query string. A query string will be add to the redirect URI, providing a code.
+
+```text
+${account.callback}?code=tQPUv...
+```
+
+Your application must then:
+
+ - handle the request, responding to the user appropriately;
+ - extract the value of the `code` parameter;
+ - call `/oauth/token` endpoint.
+ 
+Since your application will *not* have received the ID token when the user first attempts to access the callback URI, you have two choics:
+
+ - Serve an intermediary page which then redirects once the user's identity has been verified
+ - Wait to complete the request until the user's identity has been verified.
+
+In the mean time, you will need to verify the user's identy by getting an `id_token`.
+
+
+## Exchange the `code` for an `id_token`
```

Comment:
Are we exchanging it for an 'id_token'?

Reply:
That's what existing documentation says. What you get is a JSON object that includes an id_token and some other stuff.